app/main/views/frameworks.py

```diff
@@ -610,11 +607,26 @@ def upload_framework_agreement(framework_slug):
     return redirect(url_for('.framework_agreement', framework_slug=framework_slug))
 
 
-@main.route('/frameworks/<framework_slug>/signer-details', methods=['GET', 'POST'])
+@main.route('/frameworks/<framework_slug>/create-agreement', methods=['POST'])
 @login_required
-def signer_details(framework_slug):
-    framework = get_framework(data_api_client, framework_slug)
+def create_framework_agreement(framework_slug):
+    framework = get_framework(data_api_client, framework_slug, allowed_statuses=['standstill', 'live'])
```

Comment:
Also there's this odd situation where the logic that the behaviour is switched by `frameworkAgreementVersion` is spread across both the frontend and the api. Let's hope they always agree.